Lib/datetime.py

```diff
@@ -2269,7 +2269,7 @@ def fromutc(self, dt):
         raise TypeError("fromutc() argument must be a datetime instance"
                         " or None")
 
-    _maxoffset = timedelta(hours=23, minutes=59)
+    _maxoffset = timedelta(seconds=86339, microseconds=999999)
```

Comment:
should be `86399`. Or better to use hours, minutes and seconds?

Reply:
```suggestion
    _maxoffset = timedelta(hours=23, minutes=59, seconds=59, microseconds=999999)
```

I prefer the full version to make the intent clearer and to avoid further mistakes.